src/index.js

```diff
@@ -1,6 +1,8 @@
 import { isAbsolute, join } from 'path';
 
+// @ts-ignore
 import arrify from 'arrify';
+// @ts-ignore
 import micromatch from 'micromatch';
```

Comment:
You can use `import * as micromatch from 'micromatch'`, it should help

Reply:
fixed